analysis/aws_rules_guardduty/aws_guardduty_high_sev_findings.yml

```diff
@@ -0,0 +1,60 @@
+AnalysisType: rule
+Filename: aws_guardduty_high_sev_findings.py
+PolicyID: AWS.GuardDuty.HighSeverityFinding
+DisplayName: AWS GuardDuty High Severity Finding
+Enabled: true
+ResourceTypes:
```

Comment:
Shouldn't this be log type?

Reply:
@kostaspap we actually have to make the change,  the Schema is only for Policies at the moment